### ISSUE TITLE
From 'Dockerfile' not presented as option for Add Dev Container Configuration Files

### ISSUE THREAD

**Issue by dstack4273** [microsoft/vscode-remote-release#8190]
Type: <b>Bug</b>

The only way I was able to set up a dev container with an existing Dockerfile was to run the command "Open Folder in Container". The only menu options that were presented with the Add Dev Container Configuration Files were templates. If "From 'Dockerfile'" is no longer a valid option, the documentation should be updated to reflect the best starting place for someone that wishes to use an existing Dockerfile. I also didn't see the menu option for "From a predefined container configuration defintion..." 

Extension version: 0.282.0
VS Code version: Code 1.76.1 (5e805b79fcb6ba4c2d23712967df89a089da575b, 2023-03-08T16:32:00.131Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor             (24 x 3800)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|63.94GB (34.55GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by dstack4273** [comment]
I can open a separate item - but since it's tangentially related to this one I also wanted to add that this morning while working through the same process on my Ubuntu 20 box when I attempt to launch from an existing Dockerfile I get a cryptic error that I've no idea how to sort: "Command 'Dev Containers: Open Folder in Container...' resulted in an error" - Cannot read properties of undefined (reading 'localeCompare')"

**Comment by chrmarti** [comment]
We still check the workspace folder for Dockerfiles. Where is your Dockerfile located and what name does it have? Are you using a multi-root workspace?

**Comment by dstack4273** [comment]
Sorry for the delay! I had this happen from a workspace of a single folder where the Dockerfile was named Dockerfile. It was just all the default stuff that you get from the command "Docker: Add Docker Files to Workspace...". I tried it again this evening on both of my systems and it's working again as it shows in the documentation and as you say it works. Maybe I was doing something wrong to make it skip that first menu and just drop into the "from a predefined container configuration definition..." menu (which is all I was seeing the last time I tried to do this). 

But I guess since the problem isn't there anymore (or heck maybe it never was) -- I'll close this out and if it shows back up again I'll reopen and add screenshots or a screen capture or something to help with debugging.
